### DISCUSSION TITLE
Add the ability to import/convert beatmaps from other VSRG's

### DISCUSSION THREAD

**Discussion by nerootter** [ppy/osu#25948] [Ideas] [answered]
As title suggests

Would be awesome to have support of the following games:
o2Jam
BMS
Stepmania/Etterna

**Comment by peppy** [comment, marked as answer]
While I wouldn't be against a high quality implementation of this by a contributor, we won't be doing this ourselves.

Best covered by third party projects I think.

**Comment by nagi-desuuu** [comment]
There's always a D.I.Y. solution. *wink wink*
